boot/kernel_os.go

```diff
@@ -174,3 +176,49 @@ func InUse(name string, rev snap.Revision) bool {
 
 	return false
 }
+
+var ErrBootNameAndRevisionAgain = errors.New("boot revision not yet established")
+
+func GetCurrentBoot() (kernel, core *snap.Info, err error) {
```

Comment:
I'm slightly worried that somebody will be tricked into thinking that those are full Infos.

It's slightly less efficient, but given the direction we are going, it might be safer to have this already have the signature:

    ...(typ snap.Type) (name string, revision snap.Revision, err error)

Reply:
I'd rather return a `struct { KernName, KernRev, CoreName, CoreRev }` if that's not unreasonable

Reply:
or, make it a PlaceInfo and add Revision() to PlaceInfo